projects/configs/uclh-nasogastric-tube-project-ngt-only-full-dataset.yaml

```
@@ -17,7 +17,7 @@ project:
     modalities: ["DX", "CR"]
 
 tag_operation_files:
-    ["uclh-nasogastric-tube-project-ngt-only-full-dataset.yaml"]
+    base: "uclh-nasogastric-tube-project-ngt-only-full-dataset.yaml"
 
```

This will currently fail validation
```suggestion
    manufacturer_overrides: null
```